docs/configure.rst

```diff
@@ -0,0 +1,163 @@
+=====================
+Install the Croud CLI
+=====================
+
+This page shows you how to install the `Croud CLI`_. The Croud command-line
+interface (CLI) lets you issue commands to create and manage CrateDB Cloud
+organizations, projects, and resources, and deploy CrateDB clusters.
```

Comment:
I would change this to 

`This page shows you how to install the `Croud CLI`_. The Croud command-line
interface (CLI) lets you issue commands to create and manage CrateDB Cloud
organizations, projects,  CrateDB clusters and other products.`

because you can deploy a cluster, scale up and delete and you can also spin up other products such as eventhub consumers etc

Reply:
because this is now part of a larger doc called "Configure", this intro section should be merged into with the text in the next subsection "Install"

i.e., there should be a single section in this doc that is "Install the Croud CLI" with no "Install" subsection, no "Update Croud" subsection, and no command-line options reference (please stick with the link through to the croud reference for that with the SEEALSO block)

Reply:
Should we merge the "Update Croud" part into the Croud Reference? There currently doesn't seem to be an equivalent.

Reply:
if it's a part of the install process, I would include it here